docs/documentation/full-text/sorting.mdx

```diff
@@ -27,6 +27,22 @@ ORDER BY rating DESC
 LIMIT 5;
 ```
 
+When sorting by a field and a limit indexed as `fast => true`, ParadeDB will efficiently return the "top N" fields
```

Comment:
It might be nice for the `"top N"` here to hyperlink to your Top N section above.

Reply:
I'll leave that stuff up to @rebasedming.  It's in the same, very short, page already.